doc/source/docs/reference/geoseries.rst

```diff
@@ -31,6 +31,7 @@ General methods and attributes
    GeoSeries.y
    GeoSeries.z
    GeoSeries.get_coordinates
+   GeoSeries.hausdorff_distance
```

Comment:
Can you just move it right under `GeoSeries.distance` to have it a bit grouped by similarity? Thanks!